### PR TITLE
Bug 1711119 - Update Mozilla VPN to use new tables

### PR DESCRIPTION
**Pull request by relud**

### CODE REVIEW

File: mozilla_vpn/explores/subscriptions.explore.lkml [hunk 1]
```diff
@@ -235,3 +117,88 @@ view: all_subscriptions__events {
...
+    sql:
+    IF(${retention_month} = 0, 'first month', 'subsequent months');;
+  }
+
+  dimension: period_end_7_day_window {
+    hidden: yes
+    sql: GENERATE_DATE_ARRAY(${period_end_date}, ${period_end_date} + 6);;
+  }
+
+  measure: retained {
```
Comment (fbertsch) [inline mozilla_vpn/explores/subscriptions.explore.lkml:177]: Consider adding descriptions

**Review by fbertsch — APPROVED** [review]
r+wc. Most of these can probably be done as follow-up as we try out the explore

File: mozilla_vpn/explores/subscriptions.explore.lkml [hunk 2]
```diff
@@ -1,189 +1,75 @@
-include: "../views/*.view"
+include: "../views/subscriptions.view"
```
[the remark this answers is not in the document]
Comment (relud) [reply mozilla_vpn/explores/subscriptions.explore.lkml:1]: yeah, i'll move it to `views/subscriptions.view.lkml`.

Where doesn't this show syntax highlighting? It works for me in the developer console.

File: mozilla_vpn/views/funnel_fxa_login_to_protected.view.lkml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  measure: protected_count {
+    type: count
+    filters: [protected: "yes"]
+  }
+
+  measure: frac_completed_login {
+    description: "Fraction of first login attempts that completed login"
+    type: number
+    sql: ${completed_login_count}/${count};;
+  }
```
[the remark this answers is not in the document]
Comment (relud) [reply mozilla_vpn/views/funnel_fxa_login_to_protected.view.lkml:47]: okay, i moved this to custom measures and table calculations

File: mozilla_vpn/explores/subscriptions.explore.lkml (hunk 2 above)
[the remark this answers is not in the document]
Comment (fbertsch) [reply mozilla_vpn/explores/subscriptions.explore.lkml:1]: Just Github, but yeah Looker does show syntax.

**Pull request merged**